server/services/crmDisplayService.test.ts

```diff
@@ -29,38 +29,34 @@ describe('CRM Display Service', () => {
 
     expect(result).toEqual({
       sectionId: 'general-information',
+      title: 'General Information',
       subsections: [
         {
+          title: 'General Information',
           fields: [
             {
-              apiField: 'No',
               label: 'Has a previous application for an extension been made?',
+              apiField: 'hasPreviousApplication',
+              value: 'No',
```

Comment:
Can we remove apiField when returned from the crmDisplayService, as we do not use it in the template?